src/systemtest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -192,16 +192,17 @@ public void runCollectionsTest() {
         new CoverageChecker(
             options,
             "java7.util7.Collections.asLifoQueue(java7.util7.Deque) exclude",
-            "java7.util7.Collections.binarySearch(java7.util7.List, java.lang.Object) exclude",
```

Comment:
There seem to be a lot of changes to coverage goals to accommodate bloodhound.  But bloodhound is not enabled by default.  So why are these needed?  Are they only needed when bloodhound is enabled?  Should they be in the pull request, or delayed until a future pull request makes bloodhound the default (if that ever happens)?

Reply:
A subset of these changes to coverage goals were due to Bloodhound. I reverted the RandoopSystemTest class to be the same as Master and observed that a few system tests were failing even with Bloodhound disabled. I believe I have pinpointed the problem to UniformRandomSelector which makes a copy of the list of methods under test. Having a direct reference allows these tests to pass locally. I have pushed the changes and waiting to confirm that it passes on Travis as well.